_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -30,20 +30,20 @@ This rule applies to any [visible][] HTML element that is a [semantic link][], f
 
 ## Expectation
 
-For each test target, the [outcome][] of one of the following rules is "passed":
+For at least one element of the set comprised by the test target and all of its [visible][] [descendant][] elements in the [flat tree][], the [outcome][] of one of the following rules is "passed":
```

Comment:
We can't do that in a composite rule (the composite rule can't manipulate the DOM tree and doesn't know what a descendant is https://www.w3.org/TR/act-rules-format/#expectations-composite " A composite rule expectation must not use information from input aspects.")

OTOH, it's not needed. The Atomic rules are already doing that digging down the descendant, so a link will pass one atomic rule if the link or descendant is good, and the composite rule will be happy.